### ISSUE TITLE
Can't disable full start mode

### ISSUE THREAD

**Issue by nicolebrown123** [MicrosoftDocs/windows-itpro-docs#1199]
It looks like things change with the 1803 release. Without changing the configuration, the default for the Windows Kiosk is now to start as full screen. The default desktop now looks like this: https://support.microsoft.com/en-us/help/17179/windows-10-make-start-full-screen

It's not what I wanted, how I do revert it back to the way it was in previous versions where I need to click on the start button to show the tiled apps. I tried to set the FullScreenStart option to false but it doesn't seem to do anything. 

Here's my configuration:
<![CDATA[<LayoutModificationTemplate xmlns:defaultlayout="http://schemas.microsoft.com/Start/2014/FullDefaultLayout" xmlns:start="http://schemas.microsoft.com/Start/2014/StartLayout" Version="1" xmlns="http://schemas.microsoft.com/Start/2014/LayoutModification">
                      <LayoutOptions StartTileGroupCellWidth="6" FullScreenStart="false"/>
                      <DefaultLayoutOverride>
                        <StartLayoutCollection>
                          <defaultlayout:StartLayout>
                            <start:Group Name="Apps">
                              . . .
                            </start:Group>
                          </defaultlayout:StartLayout>
                        </StartLayoutCollection>
                      </DefaultLayoutOverride>
                    </LayoutModificationTemplate>
                ]]>

**Comment by nicolebrown123** [comment]
Sorry if I wasn't clear but this is about Windows 10 Kiosk using the provisioning package. I don't want it to be full screen by default.

I checked that setting and 'Use Start full screen' is Off by default, yet it is still showing it as a full screen. I tried turning it on, then off again but it makes no difference. What other setting should I check?

I tried the same package in 1709 release and it is not on full screen, which is what I want.

**Comment by shortpatti** [comment]
Just one question to determine which steps to provide...Are you using Microsoft Edge or Internet Explorer for your browser?

**Comment by nicolebrown123** [comment]
I'm not using a browser for Kiosk but followed this guide: https://docs.microsoft.com/en-us/windows/configuration/lock-down-windows-10-to-specific-apps

Thanks!

**Comment by shortpatti** [comment]
Hi @jdeckerms, since you are more familiar with the kiosk app in Windows 10, can you help @nicolebrown123 ?

**Comment by nicolebrown123** [comment]
Just to clarify, I think it's called tablet mode and not full screen mode. The documentation says:
"Lockdown profile: Users assigned a lockdown profile will see the desktop in tablet mode with the specific apps on the Start screen."

My question is: is there a way to disable this Tablet Mode and instead use the Desktop Mode just like in 1709?

**Comment by jdeckerms** [comment]
@nicolebrown123 I don't think so, but I'm checking with the product team and I'll let you know as soon as I have a definitive answer.

**Comment by jdeckerms** [comment]
@nicolebrown123 I just got confirmation that we only support tablet mode for multi-app kiosk.
